src/UI.java

```diff
@@ -77,12 +92,21 @@ private static void handleMarkMineOption() {
         System.out.println("Cell marked/unmarked as a mine.");
     }
 
-    public static void determineWinner() {
+    private static void determineWinner() {
         if (Game.getSafeCells() == 0) {
             System.out.println("Congratulations! You've won the game!");
```

Comment:
will just add this here:
`Game.revealAllCells(); ` // to get the final board view when the game is won"

Reply:
Other than this: great job 🙌👏👌
we are ready for professional jobs 💯

Reply:
I think `printBoard(true)`  with `true` argument should print all cells revealed